apps/apple/FluentUITester/.flowconfig

```diff
@@ -1,4 +1,6 @@
 [ignore]
+; We fork some components by platform
```

Comment:
Shouldn't these just not exist?

Reply:
The entire file or the reference to android? The reference to android can probably dissappear